fa.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+"TOOLBAR_PLAYBUTTON_TOOLTIP" = "پخش/توقف";
+"TOOLBAR_MUTEBUTTON_TOOLTIP" = "تغییر وضعیت سکوت";
+"TOOLBAR_VOLUMESLIDER_ACCESSIBILITY_LABEL" = "حجم صدا";
+
+"SIDEBAR_TITLE_LIBRARY" = "کتابخانه";
+"SIDEBAR_TITLE_STATIONS" = "دسته‌ها";
+"SIDEBAR_TITLE_RADIO" = "رادیو";
+
+"SIDEBAR_CATEGORY_ALL_STATIONS" = "همه ایستگاه‌ها";
+"SIDEBAR_CATEGORY_RECENTS" = "اخیر";
+
+"LIST_HEADER_STATION" = "ایستگاه";
+
+"MENU_NEW" = "جدید";
+"MENU_NEW_STATION" = "ایستگاه…";
+"MENU_NEW_GROUP" = "دسته…";
+
+"DISCOVERABILITY_HUD_NEW_STATION" = "ایستگاه جدید";
+"DISCOVERABILITY_HUD_NEW_GROUP" = "دسته جدید";
+
+"MENU_VIEW_AS_GRID" = "مربعی";
+"MENU_VIEW_AS_LIST" = "لیستی";
+
+"DISCOVERABILITY_HUD_VIEW_AS_GRID" = "نمایش به صورت مربعی";
+"DISCOVERABILITY_HUD_VIEW_AS_LIST" = "نمایش به صورت لیستی";
+
+"MENU_VIEW_SHOW_SIDEBAR" = "نمایش ستون کناری";
+"MENU_VIEW_HIDE_SIDEBAR" = "مخفی کردن ستون کناری";
+
+"MENU_HELP_GETSUPPORT" = "پشتیبانی در توییتر…";
+"MENU_HELP_SHOW_WELCOME_SCREEN" = "نمایش صفحه خوش‌آمدگویی";
+
+"PLAYBACK_ERROR_STALL" = "ارتباط با ایستگاه قطع شده است.";
+
+"SEARCH_PLACEHOLDER" = "جستجو";
+
+"ALERT_OK" = "بسیار خب";
+"ALERT_ADD" = "افزودن";
+"ALERT_CANCEL" = "انصراف";
+"ALERT_DELETE" = "حذف";
+
+"ADD_STATION_WINDOW_TITLE" = "افزودن کانال جدید";
+"EDIT_STATION_WINDOW_TITLE" = "ویرایش کانال";
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "نام";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://path/to/stream";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "انتخاب تصویر";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "افزودن دسته جدید";
+"EDIT_COLLECTION_WINDOW_TITLE" = "ویرایش دسته";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "دسته من";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "ویرایش";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "حذف دسته";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "اشتراک گذاری";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "ویرایش";
+"STATION_CONTEXT_DELETE_COLLECTION" = "حذف ایستگاه";
+"STATION_CONTEXT_MOVE" = "انتقال به دسته";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "حذف \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "از حذف این دسته و همه ایستگاه‌های داخل آن اطمینان دارید؟";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "حذف \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "آیا از حذف این دسته اطمینان دارید؟";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "بدون ایستگاه";
+
+"WELCOME_COPY" = "به Broadcasts خوش آمدید. اگر مایل باشید، ما چند ایستگاه رادیویی از محل زندگی خودمان به شما معرفی می‌کنیم تا کتابخانه‌تان را با آن آغاز کنید.\n\nیا اینکه اگر ترجیح می‌دهید به نسخه غیررایگان به روز رسانی کنید تا بتوانید ایستگاه‌های دلخواه خود را اضافه کنید یا از بین هزاران ایستگاه در فهرست نرم‌افزار انتخاب کنید.";
+
+"WELCOME_UPSELL" = “به Broadcasts خوش آمدید. اگر مایل باشید، ما چند ایستگاه رادیویی از محل زندگی خودمان به شما معرفی می‌کنیم تا کتابخانه‌تان را با آن آغاز کنید.\n\nیا اینکه اگر ترجیح می‌دهید به نسخه غیررایگان به روز رسانی کنید تا بتوانید ایستگاه‌های دلخواه خود را اضافه کنید یا از بین هزاران ایستگاه در فهرست نرم‌افزار انتخاب کنید.”;
```

Comment:
Outer quotes need adjusting here back to regular `"`

Reply:
Sure thing. That was iPad’s and Working Copy’s fault.